siliconcompiler/tools/openroad/scripts/sc_dfm.tcl

```diff
@@ -9,8 +9,10 @@ foreach obstruction [[ord::get_db_block] getObstructions] {
 }
 utl::info FLW 1 "Deleted $removed_obs routing obstructions"
 
-if { $openroad_fin_add_fill == "true" && \
-     [sc_cfg_exists pdk $sc_pdk aprtech openroad $sc_stackup $sc_libtype fill] } {
+if {
+  $openroad_fin_add_fill == "true" &&
+  [sc_cfg_exists pdk $sc_pdk aprtech openroad $sc_stackup $sc_libtype fill]
```

Comment:
I don't know why, but I find this formatting to be a little harder to read. I'm not sure what the better solution is, but I think it's because it is indent aligned with the code in the body that makes it look like it's part of that code.

Reply:
Yeah, that makes sense. The one problem is that that someone who, for example, configured a 4-space indent and no spaces in braces would get something even less readable for a similar reason:

```
if {$openroad_fin_add_fill == "true" && \
    [sc_cfg_exists pdk $sc_pdk aprtech openroad $sc_stackup $sc_libtype fill]} {
    set sc_fillrules \
         [lindex [sc_cfg_get pdk $sc_pdk aprtech openroad $sc_stackup $sc_libtype fill] 0]
```

The hope is that the extra line of `} {` helps a bit, but I get that it's not a very substantial separator.

One compromise thought I had was to cue off the existing formatting, where `tclfmt` will do the aligned style like you have if there's no newline before the start of the expression, and will do the current formatting if the user starts the expression on a new line themselves. This complicates the implementation a bit, but I can take another look at supporting it.

Reply:
I see your point. The python flake8 errors when the indent of the if and following sections align, maybe in this case the second line needs to be indented one more time?